frontend/src/container/ListOfDashboard/ImportJSON/index.tsx

```diff
@@ -239,12 +225,6 @@ function ImportJSON({
 						>
 							{t('import_and_next')} &nbsp; <MoveRight size={14} />
 						</Button>
-
-						{isFeatureAlert && (
```

Comment:
Was this not being consumed ?

Reply:
no we removed usage based features so removed the check for everywhere